src/dispatch/incident/flows.py

```diff
@@ -497,77 +497,21 @@ def incident_active_flow(incident_id: int, command: Optional[dict] = None, db_se
     # we load the incident instance
     incident = incident_service.get(db_session=db_session, incident_id=incident_id)
 
-    if incident.status == IncidentStatus.active:
-        if command:
-            convo_plugin = plugins.get(INCIDENT_PLUGIN_CONVERSATION_SLUG)
-            convo_plugin.send_ephemeral(
-                command["channel_id"],
-                command["user_id"],
-                "Incident Already Active Notification",
-                blocks=[
-                    {
-                        "type": "section",
-                        "text": {
-                            "type": "plain_text",
-                            "text": "The incident is already active. Aborting command...",
-                        },
-                    }
-                ],
-            )
-        return
```

Comment:
Are you planning to add this logic to a new function called, for example, `handle_mark_incident_acttive_command` in the views module of the Slack plugin?

Reply:
Yes, we could add it to the handle_mark_incident_active, or we could centralize it based on a signal/source.